tests/files/adapters.txt

```diff
@@ -0,0 +1,25 @@
+AAAAAAAAAAAAAAA
```

Comment:
Might need some additional files for testing adapter file validation (see comment above)